lib/Dialect/FIRRTL/Transforms/LowerIntmodules.cpp

```diff
@@ -61,6 +61,7 @@ void LowerIntmodulesPass::runOnOperation() {
   auto &ig = getAnalysis<InstanceGraph>();
 
   bool changed = false;
+  bool warningEmittedOnce = false;
```

Comment:
Mega-nit: can the variable name be more descriptive, i.e., about what warning this is?